libs/wingsdk/src/target-tf-aws/counter.inflight.ts

```diff
@@ -56,4 +57,24 @@ export class CounterClient extends CounterClientBase {
 
     return parseInt(currentValue);
   }
+
+  public async reset(value?: number): Promise<void> {
+    const command = new UpdateItemCommand({
+      TableName: this.tableName,
+      Key: { [HASH_KEY]: { S: COUNTER_ID } },
+      UpdateExpression: `SET ${VALUE_ATTRIBUTE} = :${RESET_VALUE}`,
+      ExpressionAttributeValues: {
+        [`:${RESET_VALUE}`]: { N: `${value}` },
+      },
+      ReturnValues: "UPDATED_NEW",
+    });
+
+    const result = await this.client.send(command);
+    let newValue = result.Attributes?.[VALUE_ATTRIBUTE].N;
+    if (!newValue) {
+      throw new Error(`${VALUE_ATTRIBUTE} attribute not found on table.`);
+    }
+
+    return;
```

Comment:
Since we are not trying to do any early bailout with this return I think we can just remove it.